telepresence/outbound/vpn.py

```diff
@@ -110,14 +114,17 @@ def k8s_resolve(
             continue
 
         if proxy_target in ipcache:
-            ip_ranges.append(ipcache[proxy_target])
+            if isinstance(ipcache[proxy_target], list):
+                ip_ranges += ipcache[proxy_target]
+            else:
+                ip_ranges.append(ipcache[proxy_target])
```

Comment:
This really only needs to be `ip_ranges += ipcache[proxy_target]`, but it will give exciting results if user has an old cache. Is the cache cleared if you upgrade telepresence or so?

Reply:
The cache is cleared when it is 24 hours old. That said, perhaps a better approach is to use a different cache key. So it'd be 

    cache[context]["ips"]["hostname"] -> one IP string

and

    cache[context]["ip-list"]["hostname"] -> list of IP strings

Reply:
Yes, was considering changing the key but wasn't sure if that would break something else. Couldn't really find anything related but figured I'd get feedback :) 
Changed it now!